CloudNotes/CloudNotes/MemoListCell.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  MemoListCell.swift
+//  CloudNotes
+//
+//  Created by TORI on 2021/06/03.
+//
+
+import UIKit
+
+class MemoListCell: UITableViewCell {
+    
+    let cellStackView = UIStackView()
+    let bottomStackView = UIStackView()
+    let cellTitle = UILabel()
+    let cellDate = UILabel()
+    let cellBody = UILabel()
```

Comment:
명확한 네이밍을 위해서 `cell-`로 이름을 바꾸셨군요!
```suggestion
    let titleLabel = UILabel()
    let dateLabel = UILabel()
    let bodyLabel = UILabel()
```

이런 네이밍은 어떠세요?